content/boilerplates/trace-sampling-warn.md

```diff
@@ -0,0 +1,2 @@
+When you enable tracing, you may also want to set the sampling rate that Istio will use for tracing.
```

Comment:
Warnings are a good use of boilerplates. Just be sure to include the markup needed for the callout in the boilerplate:
https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#callouts
```suggestion
{{< warning >}}
When you enable tracing, consider setting Istio's sampling rate for tracing too.
```

Reply:
Hugo errors out when `{` characters appear as the first non-whitespace characters in boilerplate.  Rather than try to engineer a workaround, I have removed that boilerplate altogether.

Reply:
There's probably an issue with using shortcodes in boilerplates, depending on when the boilerplates get expanded. I'll look into it.